lib/deliver/submit_for_review.rb

```diff
@@ -29,10 +29,19 @@ def submit!(options)
     end
 
     def select_build(options)
-      Helper.log.info "Selecting the latest build..."
       app = options[:app]
       v = app.edit_version
-      build = wait_for_build(app)
+
+      if options[:build_number]
```

Comment:
@hjanuschka what he means
```ruby
build_number = options[:build_number]
```
I don't think you need to update this, I'm ok with accessing the values directly here